### PR TITLE
finish setting up the environment

### PR DESCRIPTION
**Pull request by AdelabuAderonke**

### CODE REVIEW

File: enforcerlbp/enforcerlbp.sln [hunk 1]
```diff
@@ -1,9 +1,21 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.32228.343
+# Visual Studio Version 17
+VisualStudioVersion = 17.2.32630.192
```
Comment (afeexclusive) [inline enforcerlbp/enforcerlbp.sln:4]: Are you aware of this `VisualStudioVersion = 17.2.32630.192` change?

Comment (AdelabuAderonke) [reply enforcerlbp/enforcerlbp.sln:4]: No Sir, how do I resolve this?

Comment (afeexclusive) [reply enforcerlbp/enforcerlbp.sln:4]: Do a research on the change and find out if there is any implications then we will if we should revert or just allow it

Comment (AdelabuAderonke) [reply enforcerlbp/enforcerlbp.sln:4]: ok sir

**Pull request merged**